README.md

```diff
@@ -1,3 +1,45 @@
-# resin.io docs
+Resin Documentation
```

Comment:
I think we should be referring as "Resin.io" instead of just "Resin", since that's how the brand is registered.